src/components/reusable/card/card.ts

```diff
@@ -0,0 +1,145 @@
+import { LitElement, html } from 'lit';
+import { customElement, property, state } from 'lit/decorators.js';
+import { classMap } from 'lit/directives/class-map.js';
+
+import CardScss from './card.scss';
+
+/**
+ * Card.
+ * @slot card-logo - Slot for card logo (Required). Size 32 * 32 as per UX guidelines.
+ * @slot card-action-button - Slot for Card action button.
+ * @slot card-overflow-menu - Slot for overflow menu.
+ * @slot card-thumbnail - Slot for card thumbnail image.
+ * @slot tags - Slot for the tag component.
+ * @slot card-links - Slot for card links.
+ * @slot unnamed - Slot for card body content.
+ */
+
+@customElement('kyn-card')
+export class Card extends LitElement {
+  static override styles = CardScss;
+  /** Card width. Default 264px. */
+  @property({ type: Number })
+  cardWidth = 264;
```

Comment:
Width should be dictated by the Grid, so the card component itself should always be 100% width.

Reply:
I think maybe instead of a width prop, set a CSS width on the `:host` selector. Then it can be overridden this way:
```css
kyn-card { width: 100%; }
```
It needs to be able to use percentage width, especially within a grid.